lib/xls_function/evaluators/functions/switch.rb

```diff
@@ -0,0 +1,56 @@
+module XlsFunction
+  module Evaluators
+    module Functions
+      class Switch < ::XlsFunction::Evaluators::FunctionEvaluator
+        function_as :switch
+
+        MAX_ARGUMENTS_COUNT = 254
+        MAX_ARGUMENTS_COUNT.times do |n|
+          text_name = :"condition#{n + 1}"
+          define_arg text_name
+        end
```

Comment:
You should take efficiency into account because this function has variable arguments.
Could you try to re-consider implementing as short-circuit ? Please refer to the `ifs` function.

Reply:
I fixed it using `ifs` as a reference.
[9a6db25](https://github.com/moneyforward/xls_function/pull/16/commits/9a6db2594b13c22a745046bd9bf04be9584d618d)